airbyte-integrations/connectors/destination-jdbc/build.gradle

```diff
@@ -9,6 +9,10 @@ application {
 }
 
 dependencies {
+    implementation 'org.apache.commons:commons-lang3:3.11'
+    implementation 'org.apache.commons:commons-csv:1.4'
+    implementation 'com.github.alexmojaki:s3-stream-upload:2.2.2'
```

Comment:
It's a pretty lightweight library and is regularly updated: https://github.com/alexmojaki/s3-stream-upload
However, there aren't many public usages: https://github.com/search?q=alexmojaki+%22s3-stream-upload%22&type=code

I only used this since it was already part of @davinchia's Redshift implementation. I imagine if we do run into problems with it we can rip it out easily, so I'm not planning on changing it to something internal preemptively.